docs/modes-of-operation/_index.md

```diff
@@ -0,0 +1,136 @@
+
+Compared to terraform, the dependencies are managed in a predefined execution order and have static binding using deterministic names. Lower performance, but easier to follow. Remote state is the CR saved to the API server.
+
+### Using the reconciler mode
+
+To use One Eye in reconciler mode, complete the following steps. In this scenario, the manifest is read from a file, allowing you to declaratively provide custom configuration for the various components.
+
+1. [Login to your One Eye installation](../cli/login/).
+1. Prepare the configuration settings you want to apply in a YAML file, and run the following command. For details on the configuration settings, see the [Observer Custom Resource](#cr).
+
+    ```bash
+    one-eye reconcile --from-file <path-to-file>
+    ```
+
+1. The settings applied to the components are the result of merging the `default settings` + `valuesOverride` + `managed settings`. You cannot change the managed settings to avoid misconfiguration and possible malfunction.
+
+## Operator mode {#operator}
+
+The operator mode follows the familiar [operator pattern](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/). In operator mode, One Eye watches events on the [Observer Custom Resource](#cr), and triggers a reconciliation for all components in order, the same way you can trigger the reconcile command locally.
+
+> Note: Unlike in the [declarative CLI mode](#reconcile), in operator mode the One Eye operator is running inside Kubernetes, and not on a client machine. This naturally means that this mode is exclusive with the install, delete, and reconcile commands.
+
+![Operator mode - Declarative installer day2](/img/blog/declarative-installer/day2.png)
+
+Using the operator mode is the recommended way to integrate the One Eye installer into a Kubernetes-native continuous delivery solution, for example, Argo, where the integration boils down to applying YAML files to get the installer deployed as an operator.
+
+Existing configurations managed using the `reconcile` command work out-of-the box after switching to the operator mode.
+
+### Using the operator mode
+
+To use One Eye in operator mode, complete the following steps. In this scenario, the reconcile flow runs on the Kubernetes cluster as an operator that watches the `ControlPlane` custom resources. Any changes made to the watched custom resource triggers the [reconcile flow](#reconcile).
+
+1. [Login to your One Eye installation](../cli/login/).
+1. Run the following commands to install the One Eye operator.
+
+    ```bash
+    helm repo add banzaicloud-stable https://kubernetes-charts.banzaicloud.com/
+    helm install --name=one-eye --namespace=one-eye-system banzaicloud-stable/one-eye
+    ```
+
+1. Activate the One Eye operator on your cluster:
+
+    ```bash
+    one-eye operator reconcile
+    ```
```

Comment:
I beleive this does not work and would be interesting to understand where it came from

Reply:
In the operator docs of other products work like that, so I assumed OE uses the same pattern., I've deleted this step.